background.ts

```diff
@@ -26,6 +26,7 @@ import buildApplicationMenu from '@/main/mainmenu';
 import setupNetworking from '@/main/networking';
 import setupTray from '@/main/tray';
 import setupUpdate from '@/main/update';
+import { transientSettings, TransientSettings } from '@/store/transientSettings';
```

Comment:
The `store/` directory contains Vuex stores, which live in the renderer process.  `background.ts` is the main application entrypoint in the main process.  Please keep those separate.  For one thing, this means you have _two_ vuex stores in the application.

I'd say use `import type` instead (which is safe), but `transientSettings` is used at runtime, so that's not possible.

Reply:
I am sorry Mark @mook-as , I got your point now, thanks. Just pushed a change to eliminate the dependency.